README.adoc

```diff
@@ -99,4 +99,4 @@ spring.cloud.gcp.credentials.scopes=[SCOPE_1],[SCOPE_2],[SCOPE_3]
 These properties are optional and, if not specified, Spring Boot will attempt to automatically find them for you.
 For details on how Spring Boot finds these properties, refer to the https://cloud.spring.io/spring-cloud-gcp[documentation].
 
-NOTE: If your app is running on Google App Engine or Google Compute Engine, in most cases, you should omit the `spring.cloud.gcp.credentials.location` property and, instead, let the Spring Cloud GCP Core Starter find the correct credentials for those environments.
\ No newline at end of file
+NOTE: If your app is running on Google App Engine or Google Compute Engine, in most cases, you should omit the `spring.cloud.gcp.credentials.location` property and, instead, let the Spring Cloud GCP Core Starter find the correct credentials for those environments.
```

Comment:
What's different here?

Reply:
I used the in-browser edit to make this change, and for some reason it sometimes marked likes as being changed even though nothing actually was (which is what happened here)